form/form.go

```diff
@@ -125,18 +132,25 @@ func (f IOFiller) Fill(form Form) (map[string]interface{}, error) {
 	fields := SortedFields(form.Fields)
 	values := make(map[string]interface{}, len(fields))
 	checkers := make([]schema.Checker, len(fields))
+	allMandatory := true
 	for i, field := range fields {
 		checker, err := field.Checker()
 		if err != nil {
 			return nil, errgo.Notef(err, "invalid field %s", field.Name)
 		}
 		checkers[i] = checker
+		allMandatory = allMandatory && field.Mandatory
 	}
 	if form.Title != "" {
 		f.printf("%s\n", form.Title)
 	}
+	if allMandatory {
+		f.printf("Press return to select a default value.\n")
```

Comment:
Not sure about this. It might imply there is always a default value. Can't think if anything better in one sentence though.